changelogs/unreleased/pytest-mark-to-skip-based-on-version.yml

```diff
@@ -0,0 +1,7 @@
+---
+description: Added pytest mark to skip based on server version
+change-type: minor
```

Comment:
Why is this a minor change?

Reply:
It adds something to the api, but on the other hand, only to the testing api,...

Reply:
if you want, I can make it a patch.